.circleci/config.yml

```diff
@@ -135,6 +135,8 @@ commands:
       - run:
           name: "Install apt dependencies"
           command: |
+            echo "deb http://deb.debian.org/debian stretch-backports main" | \
+                sudo tee /etc/apt/sources.list.d/backports.list
```

Comment:
Why do we need a new CMake version?

Reply:
The version on stretch didn't support `-B`/`-S`, though in older versions there was an undocumented `-H`, but I haven't tested that out on old or new versions. But it seems like I might need to install a bit more than a CMake backport, so I might revert that part.